pkg/bench/rttanalysis/testdata/benchmark_expectations

```diff
@@ -1,57 +1,57 @@
 exp,benchmark
-10,AlterRole/alter_role_with_1_option
-13,AlterRole/alter_role_with_2_options
-17,AlterRole/alter_role_with_3_options
-10,AlterTableAddCheckConstraint/alter_table_add_1_check_constraint
-10,AlterTableAddCheckConstraint/alter_table_add_2_check_constraints
-10,AlterTableAddCheckConstraint/alter_table_add_3_check_constraints
-16,AlterTableAddColumn/alter_table_add_1_column
-16,AlterTableAddColumn/alter_table_add_2_columns
-16,AlterTableAddColumn/alter_table_add_3_columns
-14,AlterTableAddForeignKey/alter_table_add_1_foreign_key
-18,AlterTableAddForeignKey/alter_table_add_2_foreign_keys
-22,AlterTableAddForeignKey/alter_table_add_3_foreign_keys
-14,AlterTableAddForeignKey/alter_table_add_foreign_key_with_3_columns
+11,AlterRole/alter_role_with_1_option
+14,AlterRole/alter_role_with_2_options
+19,AlterRole/alter_role_with_3_options
```

Comment:
Agreed. I'll look into it.

Reply:
For some reason my branch looks up system.role_options in the namespace table, which it shouldn't do because this should be cached. Something's not quite right.

Reply:
Turns out, we were losing the system database cache after every `Reset`. This is even better. Thanks!

Reply:
Fixed, and added a regression test. This testing framework is paying off already.